From: Andrew Morton <akpm@linux-foundation.org>
Subject: mm-page_alloc-reduce-page-alloc-free-sanity-checks-checkpatch-fixes
Date: Thu Feb 23 01:48:48 PM PST 2023

WARNING: braces {} are not necessary for single statement blocks
#152: FILE: mm/page_alloc.c:956:
+	if (!IS_ENABLED(CONFIG_DEBUG_VM) && want_check_pages) {
+		static_branch_enable(&check_pages_enabled);
+	}

total: 3 errors, 2 warnings, 307 lines checked

Cc: Vlastimil Babka <vbabka@suse.cz>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/page_alloc.c |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

--- a/mm/page_alloc.c~mm-page_alloc-reduce-page-alloc-free-sanity-checks-checkpatch-fixes
+++ a/mm/page_alloc.c
@@ -953,9 +953,8 @@ void __init init_mem_debugging_and_harde
 	 * of struct pages being allocated or freed. With CONFIG_DEBUG_VM it's
 	 * enabled already.
 	 */
-	if (!IS_ENABLED(CONFIG_DEBUG_VM) && want_check_pages) {
+	if (!IS_ENABLED(CONFIG_DEBUG_VM) && want_check_pages)
 		static_branch_enable(&check_pages_enabled);
-	}
 }
 
 static inline void set_buddy_order(struct page *page, unsigned int order)
_
